log_analysis/http/empty_index.html

```diff
@@ -0,0 +1,85 @@
+<html>
+<head>
+<style>
+table {
+  font-family: arial, sans-serif;
+  border-collapse: collapse;
+  width: 100%;
+  position:absolute;
+  top: 30px;
+}
+
+td, th {
+  border: 1px solid;
+  text-align: left;
+  padding: 8px;
+}
+
+th:nth-child(odd) {
+  background-color: #dddddd;
+}
+
+#error { 
+  color: red;
+  border: black 1px solid;
+}
+</style>
+</head>
+
+<script>
+function selectElement(id, valueToSelect) {
+    let element = document.getElementById(id);
+    element.value = valueToSelect;
+}
+function hideTables() {
+   var ids = ["augur", "gunicorn", "housekeeper", "github_worker", "facade_worker", "linux_badge_worker", "pull_request_worker", "repo_info_worker", "release_worker"];
+   var i;
+   for (i = 0; i < ids.length; i ++) {
+       var elements = document.getElementsByClassName(ids[i]);
+       var j;
+       for (j = 0; j < elements.length; j++) {
+           elements[j].style.visibility="hidden";
+       }
+   }
+}
+function myFunction() {
```

Comment:
Can you give this a more descriptive function name?

Reply:
It's funny that I forgot it like that 😄